01.multithreading/MultiThreading.Task5.Threads.SharedCollection/Program.cs

```diff
@@ -0,0 +1,81 @@
+﻿/*
+ * 5. Write a program which creates two threads and a shared collection:
+ * the first one should add 10 elements into the collection and the second should print all elements
+ * in the collection after each adding.
+ * Use Thread, ThreadPool or Task classes for thread creation and any kind of synchronization constructions.
+ */
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace MultiThreading.Task5.Threads.SharedCollection
+{
+    public class Program
+    {
+        private static readonly List<int> SharedCollection = new List<int>();
```

Comment:
List will work, but next time use BlockingCollection for such Producer-Consumer tasks